src/backend/distributed/worker/worker_data_fetch_protocol.c

```
@@ -1006,78 +1009,70 @@ ForeignFilePath(const char *nodeName, uint32 nodePort, const char *tableName)
 /*
  * ExecuteRemoteQuery executes the given query, copies the query's results to a
  * sorted list, and returns this list. The function assumes that query results
- * have a single column, and asserts on that assumption. If results are empty,
- * or an error occurs during query runtime, the function returns an empty list.
- * If asUser is NULL the connection is established as the current user,
+ * have a single column and ignores other columns there are more than one. If results
+ * are empty or an error occurs during query runtime, the function returns an empty
+ * list. If asUser is NULL, the connection is established as the current user,
```

You changed the semantics here, without adjusting the comments. Previously we'd ignore errors, now we'll raise ERROR.